odd-platform-ui/src/components/Management/LabelsList/LabelEditForm/LabelEditForm.tsx

```diff
@@ -14,25 +16,24 @@ import ClearIcon from 'components/shared/Icons/ClearIcon';
 interface LabelEditFormProps {
   editBtn: JSX.Element;
   label: Label;
-  isLoading: boolean;
-  updateLabel: (params: LabelApiUpdateLabelRequest) => Promise<Label>;
 }
 
 const LabelEditForm: React.FC<LabelEditFormProps> = ({
   editBtn,
   label,
-  isLoading,
-  updateLabel,
 }) => {
-  const {
-    control,
-    handleSubmit,
-    reset,
-    formState,
-  } = useForm<LabelFormData>({
-    mode: 'onChange',
-    reValidateMode: 'onChange',
-  });
+  const dispatch = useAppDispatch();
+  const { isLoading: isDeleting } = useAppSelector(
+    getLabelDeletingStatuses
+  );
+  const { isLoading: isUpdateing } = useAppSelector(
+    getLabelUpdatingStatuses
+  );
```

Comment:
Please, rename statuses more precisely for readable code.

```suggestion
  const { isLoading: isLabelDeleting } = useAppSelector(
    getLabelDeletingStatuses
  );
  const { isLoading: isLabelUpdating } = useAppSelector(
    getLabelUpdatingStatuses
  );
```